Refactor code to move const prop to convolution 2d replacer.

kimishpatel opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #35227 Add a wrapper to wrap all optimization for mobile.
* **#35226 Refactor code to move const prop to convolution 2d replacer.**

Summary:
..

Test Plan:
python test/test_xnnpack_integration.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D20603561](https://our.internmc.facebook.com/intern/diff/D20603561)

kimishpatel committed:
Refactor code to move const prop to convolution 2d replacer.

Summary:
..

Test Plan:
python test/test_xnnpack_integration.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented on torch/csrc/jit/passes/graph_rewrite_helper.h:
```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/api/module.h>
```
what is this import?

kimishpatel replied:
It is need for `script::Module`. Recent code org change, I think took it out of ir.h. Compilation does not fail, I think, because def gets included from somewhere else.

kimishpatel replied:
Well it seems, there is no script::Module in this header file. Thanks for the catch. I will remove it.

jerryzh168 reviewed (approved):
LG

kimishpatel committed:
Update on "Refactor code to move const prop to convolution 2d replacer."


Summary:
..

Test Plan:
python test/test_xnnpack_integration.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D20603561](https://our.internmc.facebook.com/intern/diff/D20603561)

[ghstack-poisoned]

kimishpatel committed:
Update on "Refactor code to move const prop to convolution 2d replacer."


Summary:
..

Test Plan:
python test/test_xnnpack_integration.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D20603561](https://our.internmc.facebook.com/intern/diff/D20603561)

[ghstack-poisoned]

kimishpatel committed:
Update on "Refactor code to move const prop to convolution 2d replacer."


Summary:
..

Test Plan:
python test/test_xnnpack_integration.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D20603561](https://our.internmc.facebook.com/intern/diff/D20603561)

[ghstack-poisoned]
